WTWSMS/history/titles/c_alcantara.txt

```diff
@@ -12,9 +12,17 @@
 480.1.1={
 	holder = 1301295 # Salla, Visigothic dux in Spain 483
 }
-510.1.1={
-	holder=220003
+507.1.1={
+    holder = 13100027
+	liege = 0
+}
+548.1.1={
+    holder = 13100028
+}
+575.1.1={ 
+	holder=212873 #Liubagilds
+	liege = "k_visigoths"
+}
+586.1.1={ 
+	holder=36556 #Reccared
 }
-560.1.1 = {
-	holder = 1301297 # Claudius, dux Lusitaniae ?587-589
-}
```

Comment:
Why is this ruler removed? He could have it from 575 and onwards.

Reply:
I am not sure why I have done so many unnecessary things. I'll try to fix them ASAP